client/src/bin/greengrass-cli.cmd

```diff
@@ -0,0 +1,15 @@
+@echo off
+
+set SCRIPT_PATH=%~dp0
+set CLI_HOME=%SCRIPT_PATH%..
+set GGC_ROOT_PATH=%CLI_HOME%\..\..\..\..\..\..
```

Comment:
Do we need to quote these 3 values?

Reply:
I tried adding quote for *each* of the three. For some reason I don't understand, Windows is not happy when I do that.  

I also tested with a path that has space in it and it works.